systemtests/tests/strippath/testrunner-01-setup

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+
+TestName=01-setup
+
+JobName=backup-bareos-fd
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+
+# Fill ${BackupDirectory} with data.
+setup_data
+
+# Count number of directories to be stripped to fully
+# strip away $BackupDirectory prefix
+strip_count="${BackupDirectory//[!\/]}"
+strip_count="${#strip_count}"
+
+# Clear additional configs
+rm -f  "${config_directory_dir_additional_test_config}"/*
+
+# Create fileset and job with strip path setting
+cat <<END_OF_DATA > "$config_directory_dir_additional_test_config/strippath-fileset-job.conf"
+FileSet {
+  Name = "SelfTestStripPath"
+  Description = "fileset just to backup some files for selftest"
+  Include {
+    Options {
+      Signature = XXH128
+	  HardLinks = Yes
+	  Strip Path = $strip_count
+    }
+    File = "${BackupDirectory}"
+  }
+}
+Job {
+  Name = "backup-bareos-fd"
+  JobDefs = "DefaultJob"
+  Client = "bareos-fd"
+  FileSet = "SelfTestStripPath"
+}
+END_OF_DATA
```

Comment:
Im not sure I understand why those are created at runtime and arent just part of the normal configuration.